scylladb/sphinx-scylladb-theme
# Update js and python dependencies

## Context

Closes https://github.com/scylladb/sphinx-scylladb-theme/issues/262

- Updates javascript dependencies
- Updates python dependencies (except [Sphinx](https://github.com/scylladb/sphinx-scylladb-theme/issues/235) and [recommonmark](https://github.com/scylladb/sphinx-scylladb-theme/issues/168)).
- Replace `node-saas` (depecated) for `saas`.
- Clean webpack and babel unused code.

##  Releasing this change

These changes might conflict with other projects using the theme custom configuration . To ensure it doesn't affect other projects that are using the theme, I'll release a new version increasing the minor number. This will prevent projects from receiving the update automatically. 

##  How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

The terminal should not raise warning messages.

4. Open http://127.0.0.1:5500/ with your favorite browser.

The [browser's console](https://balsamiq.com/support/faqs/browserconsole/) should not warning messages.
